src/sessions.ts

```diff
@@ -125,19 +127,19 @@ export class ClientSession extends TypedEventEmitter<ClientSessionEvents> {
     super();
 
     if (topology == null) {
-      throw new MongoDriverError('ClientSession requires a topology');
+      throw new MongoInvalidArgumentError('ClientSession requires a topology');
```

Comment:
this line and the one below reference internal classes; it's not meant to be instantiated directly and I don't think the topology and server session pool are things that can be directly controlled by the user, so it's probably better typed as a runtime error